db/column_family.cc

```diff
@@ -1589,7 +1590,7 @@ ColumnFamilySet::ColumnFamilySet(const std::string& dbname,
                                  const ImmutableDBOptions* db_options,
                                  const FileOptions& file_options,
                                  Cache* table_cache,
-                                 WriteBufferManager* _write_buffer_manager,
+                                 WriteBufferManager*, /*non-used parameter*/
```

Comment:
To avoid changing the tests, we should make it fallback to old behavior if cf_options.wbm is null. In specific, `ColumnFamilySet` should still accept and store a wbm, but `ColumnFamilyData` ctor might take a different wbm depending on the specific cf_options.

Reply:
Good suggestion and done.